### ISSUE TITLE
Fluxstore crashes on startup

### ISSUE THREAD

**Issue by nadeemkhan085** [inspireui/support#3445]
Step 1 (require): let us know your detail issues & screenshots, what is the product name and version?

Step 2 (require): verify license and sharing config files via this google form - http://verify.inspireui.com (please don't share the screenshot purchase here)

*Important Note:* 
- Please help to fill the required information from the above 2 steps and don't share the public screenshot purchase here, we may close the ticket which is not included the proof of purchase. Kindly help us to create only 1-Ticket & includes all the issues, that would help us focus to resolve it better. 
- If your all ticket was resolved & closed, but would like to create a new ticket, then you don't need to submit the form again, just do simple step by linking to the previous verified ID, example: #Ticket-ID
Thank you so much for your time 😊

**Comment by nadeemkhan085** [comment]
hello, if i am running debug build apk.....every thing is going fine but app in my phone getting crush on startup its happening from last one week, before one week it was working fine

**Comment by nadeemkhan085** [comment]
flutter doctor -v
[√] Flutter (Channel stable, v1.17.1, on Microsoft Windows [Version 6.3.9600], locale en-US)
    • Flutter version 1.17.1 at C:\src\flutter\flutter
    • Framework revision f7a6a7906b (2 weeks ago), 2020-05-12 18:39:00 -0700
    • Engine revision 6bc433c6b6
    • Dart version 2.8.2

[√] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at C:\Users\admin\AppData\Local\Android\sdk
    • Platform android-R, build-tools 29.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b04)
    • All Android licenses accepted.

[√] Android Studio (version 3.6)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 45.1.1
    • Dart plugin version 192.8052
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b04)

[√] VS Code (version 1.45.1)
    • VS Code at C:\Users\admin\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.10.2

[√] Connected device (1 available)
    • Redmi 4A • 205564097d93 • android-arm64 • Android 7.1.2 (API 25)

• No issues found!

**Comment by minhcasi** [comment]
@nadeemkhan085 - there are known crash issues on Redmi OS with the altest Flutter, could you try with other devices/simulators ?

**Comment by nadeemkhan085** [comment]
I tried with Huawei device still the same....  I build again....  Still the same..  Flutter doctor - v no error help me..

**Comment by minhcasi** [comment]
@nadeemkhan085 - could you try different devices as the `Huawei` will be blocked by Google?

**Comment by minhcasi** [comment]
@nadeemkhan085 - pls have a test on latest `1.7.4` - thank you!

**Comment by nadeemkhan085** [comment]
Thank you for your support.... its working like charm... i really love the app and inspireui team.
thanks again @minhcasi for responding in such a surprising way... 
i have some questions...
build.gradle showing red line and error "can not resolve symbol GradleException" (line 25 i will attach screen shot) 
then there are some notes in red colour 
Launching lib\main.dart on CAM L21 in debug mode...
Running Gradle task 'assembleDebug'...
Note: C:\src\fluter\flutter\.pub-cache\hosted\pub.dartlang.org\firebase_admob-0.9.3+2\android\src\main\java\io\flutter\plugins\firebaseadmob\AdRequestBuilderFactory.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: C:\src\fluter\flutter\.pub-cache\hosted\pub.dartlang.org\cloud_firestore-0.13.4\android\src\main\java\io\flutter\plugins\firebase\cloudfirestore\CloudFirestorePlugin.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Note: C:\src\fluter\flutter\.pub-cache\hosted\pub.dartlang.org\uni_links-0.4.0\android\src\main\java\name\avioli\unilinks\UniLinksPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: C:\src\fluter\flutter\.pub-cache\hosted\pub.dartlang.org\url_launcher-5.4.10\android\src\main\java\io\flutter\plugins\urllauncher\WebViewActivity.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: C:\src\fluter\flutter\.pub-cache\hosted\pub.dartlang.org\video_player-0.10.11+1\android\src\main\java\io\flutter\plugins\videoplayer\VideoPlayerPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: C:\src\fluter\flutter\.pub-cache\hosted\pub.dartlang.org\webview_flutter-0.3.22+1\android\src\main\java\io\flutter\plugins\webviewflutter\FlutterWebView.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
√ Built build\app\outputs\apk\debug\app-debug.apk.
Installing build\app\outputs\apk\app.apk...

** what are these? ignore them or need to fix ?
![Screenshot (10)](https://user-images.githubusercontent.com/65464645/83377067-7e4c7680-a3e5-11ea-8a55-af16c6aa6311.png)
![Screenshot (11)](https://user-images.githubusercontent.com/65464645/83377070-7f7da380-a3e5-11ea-8414-36a1e680a46f.png)
![Screenshot (12)](https://user-images.githubusercontent.com/65464645/83377071-80163a00-a3e5-11ea-9cdb-b3488f1d2c39.png)

**Comment by minhcasi** [comment]
@nadeemkhan085 

1= yes, sometime you need to remove the app on android and rebuild again, it is happened on some Android OS, on MacOS it is running smoothly.
2= it is just the warning from Flutter SDK, don't worry about this, the app still work fine.
3= we are using Android Studio 4 and work fine (please make sure to update the Flutter plugin as well)
4= there is no problem.

We're glad that it help, please give us some feedback/rating to this page https://codecanyon.net/downloads  if the product is helpful for your business, we are working hard to update more awesome and powerful features on next coming version release. Let us know if you have any feedback. Thank you very much 😊

**Comment by nadeemkhan085** [comment]
thank you for ur fastest response..... i cant find how to give you 5 star... and write a nice review...

**Comment by nadeemkhan085** [comment]
hello 
good day 
i need support for leftside menu
1- left side menu... how to rename shop (to home) Blog (to update) and change wordpress icon
2- catagories always expend.. 
3- can i show custom catagroies? 
4- addning about us (actully webview)

for right menu 
5- where to find privacy and turms (need to put url)

to change app back ground 
6- Can i change dark mode color? means can i use #2874f0 ?

**Comment by minhcasi** [comment]
@nadeemkhan085 - thank you so much for quick response, please help to go to the download page https://codecanyon.net/downloads and select this option https://tppr.me/SjHck (under the download button)

**Comment by nadeemkhan085** [comment]
hi @minhcasi  done already

**Comment by minhcasi** [comment]
We got it, thank you so much for your feedback. Please let us know if you have any further questions 😊

**Comment by nadeemkhan085** [comment]
hello
good day
i need support for leftside menu
1- left side menu... how to rename shop (to home) Blog (to update) and change wordpress icon
2- catagories always expend..
3- can i show custom catagroies?
4- addning about us (actully webview)

for right menu
5- where to find privacy and turms (need to put url)

to change app back ground
6- Can i change dark mode color? means can i use #2874f0 ?
